### ISSUE TITLE
Backport: Air Gap Install pipeline for registry:2 images error

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#20261]
Backport: https://github.com/rancher/rancher/issues/19716

**Comment by alena1108** [comment]
The fix will be in v2.2.4-rc2

**Comment by jiaqiluo** [comment]
the bug fix is validated on Rancher: v2.2.4-rc5

Confirm that the image `registry` is tagged as `ec2-13-56-226-250.us-west-1.compute.amazonaws.com:443/rancher/registry` and pushed to the private registry.
```
$ docker images | grep registry
registry   
ec2-13-56-226-250.us-west-1.compute.amazonaws.com:443/rancher/registry
```
